### DISCUSSION TITLE
Realtime DB Updates: cost per update or eventsPerSecond?

### DISCUSSION THREAD

**Discussion by nico-k3** [supabase/supabase#12745] [Questions] [answered]
Hi! I am currently evaluating Supabase for a Project where realtime Updates of the Dashboard would be real nice. 
As I wanted to caluclate the presumable cost, I wasnt sure what metric to apply tho.
In the docs there is a "eventsPerSecond" param for the realtime options. If I were to set this to 10 and create a client that uses realtime Postgres Updates to stay on the newest version of data, would this be 10*60 = 600 Updates per minute, or just the actual number of changes to the DB Table?
Thx!

**Comment by w3b6x9** [comment, marked as answer]
@nico-k3 `eventsPerSecond` is more to help you limit the number of messages your clients send for Broadcast and Presence. It has no bearing on Postgres changes as we don't throttle that. Your connected clients will receive as many messages as your database produces.

**Reply by pierroo** [reply]
I was looking for throttling presence management as this link suggested we should use it (end of the link)
https://supabase.com/docs/guides/realtime/concepts

But your answer means there is only a way to "throttle" the sending events, not the receiving ones?

For example, in an app with thousands of simultaneously connected users, how would such a presence system work?
Wouldn't it be completely overwhelming for everyone listening to track the in and out of every other users?

For example in discord, they have a presence system where you could see in real time hundreds of people from different groups their online status: how is that manageable?
